okio/src/commonTest/kotlin/okio/BufferedSinkTest.kt

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okio
+
+import okio.ByteString.Companion.decodeHex
+import kotlin.math.pow
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.fail
+
+class CommonBufferedSinkTest {
+  class Buffer : AbstractBufferedSinkTest(BufferedSinkFactory.BUFFER)
```

Comment:
Move these lines to just be the first inside `AbstractBufferedSInkTest`? Or just put them top-level?